onnxruntime/core/common/path_string.h

```diff
@@ -16,22 +23,30 @@ using PathChar = ORTCHAR_T;
 // string type for filesystem paths
 using PathString = std::basic_string<PathChar>;
 
+inline PathString ToPathString(const PathString& s) {
+  return s;
+}
+
```

Comment:
What's the purpose of this?

Reply:
it's to allow ToPathString() to be called with std::string (and std::wstring on Windows). not ideal that it makes a copy. but it lets us write straightforward code that works on different platforms:
```c++
std::string s = "some/path";
PathString p = ToPathString(s);
```